src/pages/services/aem-cf-console-admin/api/action-bar/index.md

```diff
@@ -145,3 +145,62 @@ const guestConnection = await register({
 | Field    | Type                                                                        | Required | Description                                                                  |
 |----------|-----------------------------------------------------------------------------| ------ |------------------------------------------------------------------------------|
 | id       | `string`                                                                    | ✔️  | "id" is a "data-id" attribute of the button (please use browser developer console to find actual value). |
+
+## Examples:
+### Buttons conditional rendering
+
+In React, rendering occurs whenever the state or props of a component change. If the getButtons method is called within the rendering cycle of a component, it will be executed whenever that component is re-rendered due to state or prop changes.
+
+You can indeed use this method to dynamically return different sets of buttons based on the data that has changed on the page. For example:
+
+```js
+import { register } from "@adobe/uix-guest";
+
+// ...
+
+const guestConnection = await register({
+    id: "my.company.extension-with-action-bar-button",
+    methods: {
+        actionBar: {
+            async getButtons() {
+                const selections = await guestConnection?.host?.fragmentSelections?.getSelections?.() || [];
+                const defaultButtons = [
+                    {
+                        id: "default_btn",
+                        label: "Default Button",
+                        onClick(selections) {}
+                    }
+                ]
+
+                if (selections.length > 1 ) {
+                    return [
+                        ...defaultButtons,
+                        {
+                            id: "two_and_more_selected",
+                            label: "Ext two and more selected",
+                        }
+                    ]
+                }
+
+                return [
+                    ...defaultButtons,
+                    {
+                        id: "one_or_less_selected",
+                        label: "Ext one or less selected",
+                    }
+                ]
+            },
+        },
+    },
+});
+```
+The `defaultButtons` constant contains the buttons that will be rendered every time, doesn't meter what condition is.
```

Comment:
Please ask other team members if such a detailed code explanation is needed. I think the code is simple enough to be clear for any developer.